src/transformers/keras_callbacks.py

```diff
@@ -0,0 +1,65 @@
+from pathlib import Path
+from typing import Optional, Union
+
+from tensorflow.keras.callbacks import Callback
+
+from huggingface_hub import Repository
+from transformers.file_utils import get_full_repo_name
```

Comment:
```suggestion
from .file_utils import get_full_repo_name
```
Let's use relative imports :-)